tests/components/plugwise/test_init.py

```diff
@@ -53,5 +53,17 @@ async def test_unload_entry(hass, mock_smile_adam):
     entry = await async_init_integration(hass, mock_smile_adam)
 
     mock_smile_adam.async_reset = AsyncMock(return_value=True)
-    assert await async_unload_entry(hass, entry)
+    await hass.config_entries.async_unload(entry.entry_id)
+    await hass.async_block_till_done()
+    assert entry.state == ENTRY_STATE_NOT_LOADED
     assert not hass.data[DOMAIN]
+
+
+async def test_async_setup_entry_fail(hass):
```

Comment:
Why are all the tests changed that aren't related to the binary sensor platform?

Reply:
These were fixed as per https://github.com/home-assistant/core/pull/41473#discussion_r519464240 and https://github.com/home-assistant/core/pull/41473#discussion_r519465113 as per Chris' initial review